lib/improver/spotdata/write_output.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
```

Comment:
Probably best to just replace this with a call to `iris.save`, which will automatically save to NetCDF if given a `.nc` extension - we can add more complicated saver wrappers as needed later.